scurses/src/main/scala/net/team2xh/scurses/RichText.scala

```diff
@@ -32,7 +32,7 @@ import NoWhitespace._
  *   [/*]       Stop all
  *
  * Tags don't have to be closed
- */*/
+ */
```

Comment:
guess after removal the build will fail.
since on line 32 we have `[/*]       Stop all`

looks the project doesn't have any CI and failed build is undetectable on github.